src/main/resources/messages.properties

```diff
@@ -63,3 +65,7 @@ exception.children-amount=A user can't have more than {0} registered children
 exception.exceed.images.number=Total quantity of saved images against to the product mustn't be over {0}
 exception.advertisement-image.id.not-found=Not valid ID of the advertisement or image for update
 exception.invalid.locations.file.creating.data=Invalid data to create database initialization file for locations
+
+## Positive messages
```

Comment:
That's a thought! You can improve it adding prefix which describe whom the messages belong:
```
user.password.changed
user.email.changes
```

Reply:
will be done

Reply:
done